editor/library/quote/editor.scss

```diff
@@ -0,0 +1,4 @@
+.wp-block-quote.blocks-quote-style-2 > .editor-editable p {
```

Comment:
`.blocks-quote-style-2` shouldn't it start with editor now?

Reply:
https://github.com/WordPress/gutenberg/pull/2795#discussion_r141882622

Since these are also frontend classes too, I think we should have a separate rule for those because theme authors are likely to style them. We do not have a fixed rule for this yet, maybe `wp-block__*`, but this is a separate concern IMO

Reply:
There are a few classes that start with `.blocks-`:

![screen shot 2017-10-02 at 16 15 32](https://user-images.githubusercontent.com/699132/31081696-fc7a96c0-a78c-11e7-963b-f3b3a6cb5579.png)

Please double check all of them should stay.

Reply:
Yep, these should stay :)

Reply:
The `remove` link is a special-case, it's included in a component used in `save` and `edit` but is shown only on `edit` (via a prop). I'm going to leave it as is for now, not clear answer here.